serviceregistry-server/src/main/docker/Dockerfile

```diff
@@ -1,7 +1,14 @@
-FROM openjdk:8-jre-alpine
-MAINTAINER DIFI <espen.korra@difi.no>
+FROM openjdk:8-jre-slim
+MAINTAINER Digitaliseringsdirektoratet <servicedesk@digdir.no.>
```

Comment:
Punktum for mye på eposten her?

Reply:
Riktig. Fjernet hele linjen da det skal bli deprecated.